contracts/RewardPool/modules/APR.sol

```diff
@@ -41,7 +43,10 @@ contract APR is Initializable, AccessControl {
     }
 
     function setRSI(uint256 newRSI) public onlyRole(MANAGER_ROLE) {
-        require(newRSI <= getMaxRSI(), "TOO_HIGH_RSI");
+        if (newRSI < INITIAL_RSI_BONUS) revert TooLowRSI(newRSI, INITIAL_RSI_BONUS);
```

Comment:
Also, here you can go with only one custom error which is InvalidRSI and make both checks together